src/main/java/plugily/projects/thebridge/handlers/setup/components/ModeComponent.java

```diff
@@ -98,7 +99,9 @@ public void injectComponents(StaticPane pane) {
       new SetupInventory(arena, setupInventory.getPlayer()).openModes();
     }), 1, 0);
 
-    pane.addItem(new GuiItem(new ItemBuilder(Material.REDSTONE_TORCH)
+    Material torch = XMaterial.fromString("REDSTONE_TORCH").parseMaterial();
+
```

Comment:
Use XMaterial inline
Use "XMaterial.REDSTONE_TORCH.parseMaterial()" instead of (torch) on Line 104.

Reply:
I seriously don't know why I didn't do that